subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1113,7 +1113,9 @@ static int cmd_wifi_status(const struct shell *sh, size_t argc, char *argv[])
 		PR("DTIM: %d\n", status.dtim_period);
 		PR("TWT: %s\n",
 		   status.twt_capable ? "Supported" : "Not supported");
-		PR("Current PHY rate : %d\n", status.current_phy_rate);
+		PR("Current PHY TX rate (Mbps) : %d.%03d\n",
+		   status.current_phy_tx_rate,
```

Comment:
Got confused, as in `wifi_mgmt.h`, you defined `current_phy_tx_rate` in `Kbps`, why directly used as `Mbps`?

Reply:
should we change as
`PR("Current PHY TX rate (Mbps) : %d.%03d\n",
		   status.current_phy_tx_rate / 1000, status.current_phy_tx_rate % 1000);`?